index.d.ts

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+
+export type Position = 'top' | 'bottom' | 'left' | 'right';
+export type Trigger = 'mouseenter' | 'focus click' | 'manual';
+export type Animation = 'shift' | 'perspective' | 'fade scale' | 'none';
```

Comment:
`fade scale` should be split into two options: `'fade' | 'scale'`

Reply:
This wasn't applied to #70 and now code using `"fade"` in `animation` prop breaks